code/controllers/configuration/entries/game_options.dm

```diff
@@ -289,6 +289,8 @@
 	min_val = 0
 	max_val = 4
 
+/datum/config_entry/flag/silicon_asimov_superiority_override // Controls if Asimov Superiority appears as a perk for humans even if standard Asimov isn't the default AI lawset
```

Comment:
The doc should be above the var, and begin with `///`
https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md